digitalocean-app-platform/README.md

```diff
@@ -0,0 +1,91 @@
+# Deploying the 1Password SCIM bridge using DigitalOcean App Platform
+This document will describe how to deploy the 1Password SCIM bridge using DigitalOcean's App Platform.
+
+## Deployment Overview
+App Platform is DigitalOcean's new, fully managed solution for deploying applications via a code repository. The idea behind App platform is that there is less of
+a focus on the manual management of containers in a given environment, and more of a focus on deploying applications for immediate access. After testing
+the deployment of the SCIM bridge with App Platform, it is now being added as a deployment option for those using DigitalOcean to host the SCIM bridge.
+
+Deploying the SCIM bridge with App Platform comes with a few benefits:
+* App Platform will provide and host the URL for your SCIM bridge; you will not need to setup an A record or prepare a name for a URL as noted in [PREPARATION.md](https://github.com/1Password/scim-examples/blob/master/PREPARATION.md)
+* App Platfom will host the SCIM application for a low cost of $5/mo. An additional $5/mo or $6/mo will be utilized for the Droplet created for the Redis container.
+* There's no need to manage the container that the SCIM bridge will be running on.
+
+
+## Preparation and Deployment
+To get started with deploying the SCIM bridge using App Platform, you'll need:
+
+* Access to your organization's DigitalOcean tenant.
+* Access to your organization's Github account in order to fork this repository.
+* Access to create a Droplet for Redis in your organization's DigitalOcean tenant.
```

Comment:
Is "tenant" a technical term? Otherwise, I would sub it out for "account" in these three lines.

Reply:
what makes account "more technical" than tenant?

Reply:
It's more that `account` is more recognizable than `tenant`, at least to me. However, if the word `tenant` is used by DO itself, then we should keep it that way, for sure.

Reply:
That's one of the reasons I chose tenant; DO customers use this term quite frequently as well.

Reply:
Fair enough! Let's keep `tenant` in there. We can always revise it if customers end up confused by it, but if customers are already using that terminology it should be good to go.